src/components/Community/data.json

```diff
@@ -1,7 +1,7 @@
 {
   "hero": {
-    "pictureDesktop": "/img/community/banner.png",
-    "pictureMobile": "/img/community/banner-mobile.png",
+    "pictureDesktop": "/img/community/header_mailing_list.png",
+    "pictureMobile": "/img/community/mobile_header_mailing_list.png",
     "url": "https://www.mlprague.com/#schedule-saturday"
```

Comment:
we need to update the link .. we don't have a good one now ... @rogermparent can help you setup an anchor and some smooth scroll to the subscribe section ... or we can link to the mailchimp popup